src/RollImage.cpp

```diff
@@ -472,6 +472,8 @@ void RollImage::calculateHoleDescriptors(void) {
 //
 
 void RollImage::assignMidiKeyNumbersToHoles(void) {
+	// Initialize tracker hole -> midi number mapping
+
```

Comment:
This comment should be removed, I think?

Reply:
There's no compelling reason to keep it, I guess.

Reply:
It looks like it was superseded by the one at line 793?

Reply:
Yeah... it's also kind of inaccurate here; the tracker hole -> midi map is initialized elsewhere, as you note; the purpose of this function, as the name suggests, is to assign MIDI key values to every hole *in the full punch matrix for the roll.* Those values are assigned and used in the process of MIDI data output, though curiously they aren't used when the hole report is generated because that happens prior to MIDI generation, hence the need for a bidirectional tracker hole <-> midi key mapping...